### ISSUE TITLE
Unable to visualise model gradients in 0.8.2

### ISSUE THREAD

**Issue by aleSuglia** [allenai/allennlp#2600]
**Describe the bug**
I'm training a custom model that I've written. In previous AllenNLP versions I was able to see the `gradients_mean`, `gradients_std` and `gradient_norm` in the Tensorboard. I don't know why I can't see them anymore after the recent update. Is anything changed?

**To Reproduce**

1. Train model using the `train` command
2. Start tensorboard using the command `tensorboard --logdir my_models_path` 

**Expected behavior**
All the metrics and all the gradients information should be available.

**System (please complete the following information):**
 - OS: Linux/MacOS
 - Python version: Python 3.7.2 (Anaconda)
 - AllenNLP version: 0.8.2 installed via pip
 - PyTorch version: 1.0.1.post2
